# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../libmpeg3/stray-backslash.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- libmpeg3-1.6_orig/Makefile	2005-08-28 05:21:13.000000000 +0200
+++ libmpeg3-1.6/Makefile	2006-04-11 16:39:54.086236500 +0200
@@ -146,7 +146,7 @@
 
 LIBS = -lm -lpthread
 
-$(shell if ! test -d $(OBJDIR) \; then mkdir -p $(OBJDIR) \; fi )
+$(shell if ! test -d $(OBJDIR) ; then mkdir -p $(OBJDIR) ; fi )
 
 $(shell echo $(CFLAGS) > $(OBJDIR)/c_flags)
 $(shell echo $(A52CFLAGS) > $(OBJDIR)/a52_flags)
